docs/WHOLE_STUDY_IMPORT.md

```diff
@@ -0,0 +1,204 @@
+  + `display_name`: This is what will show as a long form title on the site home page
+  + `short_name`: This is the short version. By default, should be the same as `cancer_study_identifier`
+  + `file_loc_defs`: This is where the tool paths/reference paths are located.
+
+
+## Starting file inputs
+Most starting files are exported from the D3b Warehouse. An example of file exports can be found here `cbioportal_etl/scripts/export_clinical.sh`, we now use `cbioportal_etl/scripts/get_study_metadata.py` to get the file locations.
+The generated `study_config.json` is recommended to use for each study.
+
+### cbioportal_etl/scripts/get_study_metadata.py
+```
+usage: get_study_metadata.py [-h] [-db DB_INI] [-p PROFILE] [-sc STUDY_CONFIG] [-r REF_DIR] [-a]
+
+Pull clinical data and genomics file etl support from D3b data warehouse.
+
+options:
+  -h, --help            show this help message and exit
+  -db DB_INI, --db-ini DB_INI
+                        Database config file - formatting like aws or sbg creds
+  -p PROFILE, --profile PROFILE
+                        ini profile name
+  -sc STUDY_CONFIG, --study-config STUDY_CONFIG
+                        json config file with study information; see cbioportal_etl/STUDY_CONFIGS/json_files for examples
+  -r REF_DIR, --ref-dir REF_DIR
+                        dir name containing template data_clinical* header files
+  -a, --all             flag to include all relevant files, not just status=active files, NOT RECOMMENDED
+```
+
+### From D3b Warehouse
+#### - Data clinical sample sheet
+This is the cBioPortal-formatted sample sheet that follows guidelines from [here](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#clinical-sample-columns)
+
+#### - Data clinical patient sheet
+This is the cBioPortal-formatted patient sheet that follows guidelines from [here](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#clinical-patient-columns)
+
+
+## Download starting file inputs 
+### cbioportal_etl/scripts/get_files_from_manifest.py
+```
+usage: get_files_from_manifest.py [-h] [-m MANIFEST] [-f FTS] [-at AWS_TBL] [-sp SBG_PROFILE] [-c CBIO] [-ao] [-rm] [-d] [-o]
+
+Get all files for a project.
+
+options:
+  -h, --help            show this help message and exit
+  -m MANIFEST, --manifest MANIFEST
+                        csv list of of genomic file location manifests
+  -f FTS, --file-types FTS
+                        csv list of workflow types to download
+  -at AWS_TBL, --aws-tbl AWS_TBL
+                        Table with bucket name and keys to subset on
+  -sp SBG_PROFILE, --sbg-profile SBG_PROFILE
+                        sbg profile name. Leave blank if using AWS instead
+  -c CBIO, --cbio CBIO  Add cbio manifest to limit downloads
```

Comment:
I updated this recently in the tool to warn that, if cbio_file_name_id file is used as manifest to NOT provide it again.